src/client/CodeEditor/json1PresenceDisplay.ts

```diff
@@ -6,14 +6,20 @@ import {
 } from '@codemirror/view';
 import { Annotation, RangeSet } from '@codemirror/state';
 import ColorHash from 'color-hash';
-import { Username } from '../../types';
+import { FileId, Username } from '../../types';
+import { useCallback, useContext, useMemo, useState } from 'react';
+import { TabState, VZAction } from '../vzReducer';
+import { VZCodeContext } from '../VZCodeContext';
+import { useActions } from '../useActions';
 
 const debug = false;
 
 // TODO: Make this a setting in the UI.
 // See https://github.com/vizhub-core/vzcode/issues/739
-const enableAutoFollow = false;
-
+export let enableAutoFollow = true;
+export const toggleAutoFollowButton = () => {
```

Comment:
I changed all this around in https://github.com/vizhub-core/vzcode/pull/769

It may be best to try building this feature fresh from `main` at this point.